osmaxx/excerptexport/models/excerpt.py

```diff
@@ -84,6 +85,13 @@ def color(self):
     def extent(self):
         return self.bounding_geometry.extent
 
+    @property
+    def has_running_exports(self):
+        return any([not i.can_be_deleted for es in self.extraction_orders.all() for i in es.exports.all()])
```

Comment:
It seems illogical to have `has_running_exports` (something that can be decided objectively) depend on `Export.can_be_deleted` (the rules for which we've decided subjectively). Also, if the rules of the latter ever change, the results of this new property will become wrong (unless we remember to adapt it).

Suggestion: Rename `Export.can_be_deleted` and alias it as `Export.can_be_deleted`.